### PR TITLE
Add CodeMirror syntax highlighting to form fields

### PR DESCRIPTION
**Pull request by bradp**
Adds CodeMirror functionality from the theme/plugin file edits into our fields.

Screenshot 2020-06-09 12 46 34

### CODE REVIEW

File: ihaf.php [hunk 1]
```diff
@@ -142,7 +143,28 @@ function adminPanel() {
...
     	// Load Settings Form
         include_once( $this->plugin->folder . '/views/settings.php' );
-    }
+ 	}
+
+ 	/**
+ 	 * Enqueue and initialize CodeMirror for the form fields.
+ 	 */
+ 	function initCodeMirror() {
+ 		// Enqueue code editor and settings for manipulating HTML.
```
Comment (bradp) [inline ihaf.php:152]: This needs to run only on our settings page

**Comment by bradp** [comment]
> Unless I'm missing it, this would throw errors on WP < 4.9 since the `wp_enqueue_code_editor` was not introduced until then.

Added a check for this.

File: ihaf.php [hunk 2]
```diff
@@ -142,7 +143,40 @@ function adminPanel() {
...
+ 	 */
+ 	function initCodeMirror() {
+ 		// Make sure that we don't fatal error on WP versions before 4.9.
+ 		if ( ! function_exists( 'wp_enqueue_code_editor' ) ) {
+ 			return;
+ 		}
+
+ 		global $pagenow;
+
+ 		if ( ! ( $pagenow == 'options-general.php' && isset( $_GET['page'] ) && $_GET['page'] == 'insert-headers-and-footers' ) ) {
```
Comment (peterwilsoncc) [inline ihaf.php:159]: Nits: `===` (I'll deal with this)

**Review by peterwilsoncc — CHANGES_REQUESTED** [review]
LGTM, labeling with change request so I remember to deal with my nits.

**Review by peterwilsoncc — APPROVED** [review]

**Pull request merged**